aws-lambda-java-tests/src/test/resources/sqs/sqs_event_product.json

```diff
@@ -0,0 +1,22 @@
+{
+  "Records": [
+    {
+      "messageId": "d9144555-9a4f-4ec3-99a0-34ce359b4b54",
+      "receiptHandle": "13e7f7851d2eaa5c01f208ebadbf1e72==",
+      "body": "{\n  \"id\": 1234,\n  \"name\": \"Playstation 5\",\n  \"price\": 450\n}",
```

Comment:
Maybe this is just me, but I'd rather we not mention the name of real-life products in code

Reply:
sure!